rabbit-escape-engine/src/rabbitescape/engine/textworld/LineProcessor.java

```diff
@@ -155,18 +155,17 @@ public String metaString( String key, String def )
         }
         else
         {
-            ArrayList<String> ret = new ArrayList<String>( temp.size() );
+            String[] ret = new String[temp.size()];
+            for ( int i = 0; i < temp.size() ; i++ )
+            {
+                ret[0] = "" ;
```

Comment:
It should. the fact it works with that error makes me wonder if setting the strings to "" can be skipped.